docker-dist/Dockerfile

```diff
@@ -19,29 +19,22 @@ FROM jboss/wildfly:10.1.0.Final
 
 MAINTAINER Heiko W. Rupp <hawkular-dev@lists.jboss.org>
 
-ADD target/installer.jar /tmp/
 # ADD test-simple.war /opt/jboss/wildfly/standalone/deployments/
-ADD src/main/resources/standalone.conf /opt/jboss/wildfly/bin
+ADD target/hawkular-javaagent-wildfly-dist/bin/standalone.conf $JBOSS_HOME/bin
+ADD target/hawkular-javaagent-wildfly-dist/bin/hawkular-javaagent.jar $JBOSS_HOME/bin
+ADD target/hawkular-javaagent-wildfly-dist/standalone/configuration/hawkular-javaagent-config.yaml $JBOSS_HOME/standalone/configuration/
 
+ENV HAWKULAR_SERVER_PROTOCOL http
 ENV HAWKULAR_SERVER_ADDR hawkular
 ENV HAWKULAR_SERVER_PORT 8080
 ENV HAWKULAR_AGENT_USER=jdoe HAWKULAR_AGENT_PASSWORD=password
-ENV JAVA_OPTS="-Xmx256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true -XX:MaxMetaspaceSize=256m"
+ENV JAVA_OPTS="-Xmx256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman,org.jboss.logmanager -Djava.awt.headless=true -XX:MaxMetaspaceSize=256m"
```

Comment:
Ok, thought so, but was confused because you put the logmanager here and not in standalone.conf.
I think it may be good to add this in standalone.conf with some check if it was set before, as the opts that are passed here come from env and the user could do `docker -e JAVA_OPTS=frobnitz` and thus lose the logmanager , which may end up with strange errors